libkbfs/data_types.go

```diff
@@ -195,6 +226,10 @@ const (
 	// FilesWithHolesDataVer is the data version for files
 	// with holes.
 	FilesWithHolesDataVer DataVer = 2
+	// BigFilesDataVer is the data version for files that have
```

Comment:
since `DataVer` is a per-block attribute, the comments should reflect that. Perhaps:

```
FilesWithHolesDataVer is the data version for blocks with holes.
...
BigFilesDataVer is the data version for blocks that have multiple levels of indirection, i.e. for indirect blocks that point to other indirect blocks.
```

Maybe the names should change too -- perhaps `BlockWithHolesDataVer` and `BigBlockDataVer`. Maybe even drop the `DataVer` suffix.